swagger_parser/lib/src/parser/utils/type_utils.dart

```diff
@@ -121,19 +125,31 @@ String? protectDefaultValue(
 }
 
 /// Protect enum items names from incorrect symbols, keywords, etc.
-Set<UniversalEnumItem> protectEnumItemsNames(Iterable<String> names) {
+Set<UniversalEnumItem> protectEnumItemsNames(Map<String, dynamic> enumMap) {
   var counter = 0;
   final items = <UniversalEnumItem>{};
+  final names = enumMap['enum'] as List<dynamic>;
```

Comment:
Instead of writing toString everywhere, you can convert this list to a list of strings.

Reply:
This wasn't my code, but I've mapped it to a String set now and removed the use of toString everywhere.